server/controllers/authController.js

```diff
@@ -114,4 +112,35 @@ module.exports = {
     const payload = { success: false };
     return res.status(422).send(payload);
   },
+  async logout(req, res) {
```

Comment:
We should validate that the token has a valid signature before revoking it as well!

Reply:
good call! -- this is reflected in #11